test/export/data/currencies_test.rb

```diff
@@ -36,7 +36,12 @@ class TestCldrCurrencies < Test::Unit::TestCase
     currencies = Cldr::Export::Data::Currencies.new('de')[:currencies]
     assert_empty codes - currencies.keys, "Unexpected missing currencies"
     assert_empty currencies.keys - codes, "Unexpected extra currencies"
-    assert_equal({ :one => 'Euro', :other => 'Euro', :symbol => '€' }, currencies[:EUR])
+    assert_equal({ :name => 'Euro', :one => 'Euro', :other => 'Euro', :symbol => '€' }, currencies[:EUR])
```

Comment:
Maybe we should add an assertion for a currency whose values differ. Maybe JPY?
```ruby
assert_equal({ :name => 'Japanischer Yen', :one=>'Japanischer Yen', :other=>'Japanische Yen', :symbol=>'¥' }, currencies[:JPY])
```

Reply:
I've generally treated the CLDR data as "correct" in the sense that there's not too much we as downstream consumers can do if it's wrong. Moreover, I'm fairly sure currencies get added and removed on a semi-frequent basis, so I'm not sure what value a test like this would bring us considering it would break whenever a currency like the South Sudanese pound is added.